fr.openfisca.org/api/v22/deploy.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -ex
+
+REQUIREMENTS="/home/openfisca/openfisca-ops/fr.openfisca.org/api/v22/requirements.txt"
+
+source /home/openfisca/virtualenvs/api-fr22/bin/activate
+pip install --requirement "$REQUIREMENTS" --upgrade
+# The current user must have been specifically allowed to run the next command
+# Use the visudo command to do so
+sudo systemctl restart openfisca-web-api-fr22.service
```

Comment:
`openfisca` must be given the right to run this command with `sudo`. You can do that by `sudo visudo`.
This should probably be documented in [this section](https://github.com/openfisca/openfisca-ops#serve-a-new-version-of-the-public-api).